### PR TITLE
 Created [html,css] For Seller Page

### PR DESCRIPTION
**Pull request by riyadsalem**
Release #3 is Completed

### CODE REVIEW

File: seller/seller.html [hunk 1]
```diff
@@ -32,7 +32,6 @@
           <img src="img/UntitledDesign.png" alt="logo Website" />
```
Comment (dev-mohmmed-alshorafa) [inline seller/seller.html]: remove this

File: seller/seller.html [hunk 2]
```diff
@@ -32,7 +32,6 @@
...
         </a>
       </div>
-      <i class="fas fa-bars" id="btn-close-navbar"></i>
 
       <!-- navBar Content -->
       <div id="navbar-content" class="navbar-content navbar-hidden">
```
Comment (dev-mohmmed-alshorafa) [inline seller/seller.html]: change div to nav

File: seller/seller.html [hunk 3]
```diff
@@ -54,16 +53,171 @@
...
+    <!-- Modal For Add Products -->
+    <section
+      class="modal-create-product modal-hidden"
+      id="modal-create-product"
+    >
+      <div class="modal-product-content">
+        <h2 id="add-product-title" class="add-product-title">
+          Add more products to sell more
+        </h2>
+        <div class="add-product-content">
```
Comment (dev-mohmmed-alshorafa) [inline seller/seller.html:62]: change div to form

File: seller/seller.html [hunk 4]
```diff
@@ -54,16 +53,171 @@
...
+
+              <button id="btnGrid" class="btn grid">
+                <i class="far fa-border-all active"></i>
+              </button>
+            </section>
+          </section>
+        </section>
+
+        <!-- Item For Cart -->
+        <section id="product-list" class="wrap-product-list">
```
Comment (dev-mohmmed-alshorafa) [inline seller/seller.html]: change section to figure

File: seller/seller.html [hunk 5]
```diff
@@ -54,16 +53,171 @@
...
+                <div data-index="0" class="edit">
+                  <i class="fa fa-pencil"></i>
+                </div>
+                <div class="category">Animals</div>
+                <img
+                  class="img"
+                  src="https://greenbackend.com/static/media/gbe-hero.54f456ed.png"
+                />
+              </div>
+              <div class="info-product">
```
Comment (dev-mohmmed-alshorafa) [inline seller/seller.html:197]: change div to figcaption

**Review by dev-mohmmed-alshorafa — CHANGES_REQUESTED** [review]

**Comment by riyadsalem** [comment]
#3 is Completed.

**Review by dev-mohmmed-alshorafa — APPROVED** [review]

**Review by bayan2002 — APPROVED** [review]

**Pull request merged**